src/components/map/LocationMarker.jsx

```diff
@@ -1,22 +1,24 @@
-import React, { Component } from "react";
-import { Marker, InfoWindow } from "react-google-maps";
-import "./LocationMarker.css";
+import React, { Component } from 'react';
+import { Marker, InfoWindow } from 'react-google-maps';
+import './LocationMarker.css';
 import Button from '../button';
-import { withRouter } from 'react-router-dom'
+import { withRouter } from 'react-router-dom';
 
 class LocationMarker extends Component {
-
-  constructor(props){
+  constructor(props) {
     super(props);
     this.handleYesClick = this.handleYesClick.bind(this);
+    this.onToggleInfo = this.onToggleInfo.bind(this);
   }
 
-  onToggleInfo = () => {
```

Comment:
Ah right, eslint on streetlives-web doesn't like class properties for some reason... I wonder why that is; it doesn't mind them on streetlive-api...